lib/screens/settings/settings.dart

```diff
@@ -82,6 +82,14 @@ class _SettingsScreenState extends State<SettingsScreen> {
               Navigator.pushNamed(context, "/settings/third-party");
             },
           ),
+          SwitchListTile.adaptive(
+            title: const Text('Show viewer and follower count'),
+            subtitle: const Text('Disable/Enable across the app'),
```

Comment:
I don't think the subtitle is necessary.

Reply:
Removed.